frontend/src/app/pages/exercises/exercise/shared/exercise-settings/exercise-settings-modal/exercise-settings-modal.component.html

```diff
@@ -49,10 +49,23 @@ <h4 class="modal-title">Einstellungen</h4>
                 ></app-display-validation>
             </div>
         </div>
+        <button (click)="updateTileMapProperties()" class="btn btn-primary">
+            Speichern
+        </button>
     </div>
-
     <br />
-    <button (click)="updateTileMapProperties()" class="btn btn-primary">
-        Speichern
-    </button>
+    <div class="card">
+        <div class="card-header">Übungseinstellungen</div>
```

Comment:
Maybe rather something like `Übungsmodus` - to better differentiate between the tile server and this?